lerna-package-lock.json

```diff
@@ -40703,7 +40703,7 @@
 		"isarray": {
 			"version": "0.0.1",
 			"resolved": "https://registry.npmjs.org/isarray/-/isarray-0.0.1.tgz",
-			"integrity": "sha512-D2S+3GLxWH+uhrNEcoh/fnmYeP8E8/zHl644d/jdA0g2uyXvy3sb0qxotE+ne0LtccHknQzWwZEzhak7oJ0COQ=="
+			"integrity": "sha1-ihis/Kmo9Bd+Cav8YDiTmwXR7t8="
```

Comment:
This commit doesn't do anything other than change the integrity to sha1. @curtisman I've run into this issue before as well using codespaces (which I think Ji is also using). What is the npm version that we're supposed to be using? I wonder if there's a mismatch between the recommendation and the codespaces configuration. Codespaces is running npm v6.14.17 and node v14.20.0.